packages/contentstack-config/src/commands/config/set/region.ts

```diff
@@ -46,6 +46,7 @@ export default class RegionSetCommand extends BaseCommand<typeof RegionSetComman
     '$ csdx config:set:region EU',
     '$ csdx config:set:region AZURE-NA',
     '$ csdx config:set:region AZURE-EU',
+    '$ csdx config:set:region GCP-NA',
```

Comment:
@cs-raj This example should not be there as part of current release

Reply:
Removed